test/file.jl

```diff
@@ -108,12 +108,12 @@ if is_windows()
 else
     @test filesize(dir) > 0
 end
-now = time()
+nowtime = time()
 # Allow 10s skew in addition to the time it took us to actually execute this code
-let skew = 10 + (now - starttime)
+let skew = 10 + (nowtime - starttime)
     mfile = mtime(file)
     mdir  = mtime(dir)
-    @test abs(now - mfile) <= skew && abs(now - mdir) <= skew && abs(mfile - mdir) <= skew
+    @test abs(nowtime - mfile) <= skew && abs(nowtime - mdir) <= skew && abs(mfile - mdir) <= skew
```

Comment:
Notice the commit date was actually in september, so this isn't technically necessary for `make testall1` to pass any more since your PR, but at the time I authored this it was. And may as well still do it to avoid shadowing.